### PR TITLE
Add Enum inside app\Enums Directory

### PR DESCRIPTION
**Pull request by dipesh79**
When making enum using artisan command, it add the enum inside ```app``` directory and this PR makes it inside ```app/Enums``` directory

### CODE REVIEW

**Comment by crynobone** [comment]
the command already generate to `app/Enums` if the directory exists.

**Comment by Dipesh79** [comment]
@crynobone It will only if the Enums directory is available.

File: src/Illuminate/Foundation/Console/EnumMakeCommand.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ protected function getDefaultNamespace($rootNamespace)
         return match (true) {
             is_dir(app_path('Enums')) => $rootNamespace.'\\Enums',
             is_dir(app_path('Enumerations')) => $rootNamespace.'\\Enumerations',
-            default => $rootNamespace,
+            default => $rootNamespace.'\\Enums',
```
Comment (shaedrich) [inline src/Illuminate/Foundation/Console/EnumMakeCommand.php:75]: Aren't `is_dir(app_path('Enums'))` and `default` now redundant?

**Comment by shaedrich** [comment]
> the command already generate to `app/Enums` […] [but] only if the Enums directory is available.

Yeah, that seems weird. Why not just create the directory 🤷🏻 Would have been my logical conclusion ("principle of least surprise") as well.

Isn't that what is done for other classes in Laravel?

Also, it seems weird so have to check this via `match` instead of putting `"Enums/"` in some config file as a default, so it can be changed there if needed.

**Pull request closed without merging**